nodeApps/createPrivateRelease.sh

```diff
@@ -88,17 +112,25 @@ echo "Latest commit is: ${CURRENT_SHA}"
 # Tag this deploy and make a release in github
 RELEASE_JSON="{\"tag_name\": \"v${THIS_VERSION}\",\"target_commitish\": \"${CURRENT_SHA}\",\"name\": \"v${THIS_VERSION}\",\"body\": \"${MESSAGE}\",\"draft\": false,\"prerelease\": false}"
```

Comment:
Was also nice to have the link back to the build from the release... nice easy way to create an audit trail